5.3c.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import time
+import math
+"""Defining the matrices"""
+C=np.array([[1,2,3,1,3],[2,3,4,5,5],[4,1,6,2,7],[1,2,6,1,9],[3,2,1,5,6]])
+print("The first matrix is:")
+print(C)
+D=np.linalg.inv(C)
+print("Its Inverse matrix is:")
+print(D)
+
+def Nested_For(A,B):
+    """Using nested for"""
+    start= time.time()
+    result=np.zeros([5,5])
+    for i in range(len(A)):
+        for j in range(len(B[0])):
+            for k in range(len(B)):
+                result[i][j] += A[i][k] * B[k][j]
+    result[0][0]=math.ceil(result[0][0])
+    for r in result:
+        print(r.astype(int))
+    end=time.time()
```

Comment:
It is better to take the timestamp before the print() so that the print overhead is not included